keanu-project/src/main/java/io/improbable/keanu/tensor/INDArrayShim.java

```diff
@@ -24,76 +24,116 @@
  */
 public class INDArrayShim {
 
-    public static void muli(INDArray left, INDArray right, INDArray result) {
```

Comment:
Good change - this looks like it brings us more in line with the Base API anyway.